build/containers/Dockerfile.agent-full

```diff
@@ -0,0 +1,17 @@
+ARG PLATFORM=amd64
+ARG CROSS_BUILD_TARGET=x86_64-unknown-linux-gnu
+FROM ${PLATFORM}/debian:buster-slim
+ARG CROSS_BUILD_TARGET
+RUN echo "Creating container based on ${PLATFORM}/debian:buster-slim"
+RUN echo "Using Rust binaries from ${CROSS_BUILD_TARGET}"
+
+# Link the container to the Akri repository
+LABEL org.opencontainers.image.source https://github.com/deislabs/akri
+
+# Copy over container legal notice
+COPY ./build/container-images-legal-notice.md .
+
+RUN apt-get update && apt-get install -y --no-install-recommends libssl-dev openssl && apt-get clean
+COPY ./target/${CROSS_BUILD_TARGET}/release/agent-full /agent
+ENV RUST_LOG agent,akri_shared
```

Comment:
does the component name change when we build agent-full?  should RUST_LOG container `agent_full`?

Reply:
No it shouldn't. I am simply moving the `agent` binary to `agent-full`. The contents, including the rust package names should be the same